### PR TITLE
Using actual metrics instead of stage scheduling info to determine the effective usage

### PR DESCRIPTION
**Pull request by sundargates**
### Context

^^^

### Checklist

- [ ] `./gradlew build` compiles code correctly
- [ ] Added new tests where applicable
- [ ] `./gradlew test` passes all tests
- [ ] Extended README or added javadocs where applicable

### CODE REVIEW

**Review by calvin681 — APPROVED** [review]

File: mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/JobAutoScaler.java [hunk 1]
```diff
@@ -271,80 +273,25 @@ protected Map<Integer, ClutchConfiguration> getClutchConfiguration(String jsonCo
...
+      int stage;
+      double value;
+      double effectiveValue;
+      int numWorkers;
+      String message;
+
+        public Event(ScalingReason type, int stage, double value, double effectiveValue, int numWorkers) {
+            this.type = type;
+            this.stage = stage;
+            this.value = value;
```
Comment (fdc-ntflx) [inline mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/JobAutoScaler.java:290]: do we still need value? why not just effectiveValue?

File: mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/control/AdaptiveAutoscaler.java [hunk 2]
```diff
@@ -1,64 +0,0 @@
-/*
```
Comment (fdc-ntflx) [inline mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/control/AdaptiveAutoscaler.java:1]: removed because not used anywhere?

**Review by fdc-ntflx — APPROVED** [review]

File: mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/control/AdaptiveAutoscaler.java (hunk 2 above)
Comment (sundargates) [reply mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/control/AdaptiveAutoscaler.java:1]: yup!

File: mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/JobAutoScaler.java (hunk 1 above)
Comment (sundargates) [reply mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/JobAutoScaler.java:290]: It is still being used in some places.

File: mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/JobAutoScaler.java [hunk 3]
```diff
@@ -271,80 +273,25 @@ protected Map<Integer, ClutchConfiguration> getClutchConfiguration(String jsonCo
...
+            this.effectiveValue = effectiveValue;
+            this.numWorkers = numWorkers;
+            this.message = "";
```
Comment (Andyz26) [inline mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/JobAutoScaler.java:293]: is this field ever used?

**Review by Andyz26 — APPROVED** [review]

**Pull request merged**